src/AspNetCoreModuleV2/RequestHandlerLib/AppOfflineTrackingApplication.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#include "stdafx.h"
+#include "AppOfflineTrackingApplication.h"
+#include "EventLog.h"
+#include "exceptions.h"
+
+extern HANDLE g_hEventLog;
+
+HRESULT AppOfflineTrackingApplication::StartMonitoringAppOffline()
+{
+    LOG_INFOF("Starting app_offline monitoring in application %S", m_applicationPath.c_str());
```

Comment:
WLOG_INFOF? m_applicationPath is a wstring.

Reply:
Capital S, we need to drop `WLOG_INFOF`